client_test.go

```diff
@@ -31,8 +31,11 @@ import (
 	"github.com/pkg/errors"
 	"github.com/stretchr/testify/assert"
 
+	"fmt"
+
 	dosaRenamed "github.com/uber-go/dosa"
-	"github.com/uber-go/dosa/connectors/devnull"
+	_ "github.com/uber-go/dosa/connectors/devnull"
```

Comment:
Because this has the side effect of calling the init() method in the devnull package, if it hasn't already been called, which registers the devnull connector so it can be fetched by name later.